### PR TITLE
Chromium does not support `DedicatedWorkerGlobalScope.requestAnimationFrame()` in nested workers

### LINKED ISSUE #21964: DedicatedWorkerGlobalScope.requestAnimationFrame - Not supported in nested Web Workers on Chromium
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

My testing has shown that requestAnimationFrame functionality within nested web workers on Chromium browsers is not yet supported. Chromium browsers should instead be marked as partially compatible with this caveat for now.

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

The functionality should work in accordance with the compatibility data.

### Did you test this? If so, how?

Here is the code snippet I used to test for this functionality:

`index.html`
```html
<script>
  new Worker('worker1.js').onmessage = (e) => {
    if (e.data === 'worker1 works')
      document.write("first worker raf works<br>");
    else if (e.data === 'worker2 works')
      document.write("nested worker raf works");
  }
</script>
```
`worker1.js`
```js
requestAnimationFrame(() => self.postMessage("worker1 works"));
new Worker('worker2.js').onmessage = () => self.postMessage("worker2 works");
```
`worker2.js`
```js
requestAnimationFrame(() => postMessage(undefined));
```

When it fails, it errors:
```
Uncaught DOMException: Failed to execute 'requestAnimationFrame' on 'DedicatedWorkerGlobalScope': requestAnimationFrame not supported in this Worker.
```

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

The [current compatibility data](https://github.com/mdn/browser-compat-data/blob/9e245478a79c555698cfaf2df9ba1b05fef264c6/api/DedicatedWorkerGlobalScope.json#L344), the [spec](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#animationframeprovider), and the [MDN page](https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/requestAnimationFrame).

Paraphrasing the spec, it mentions that one way requestAnimationFrame is considered supported is if its associated web worker initiator also supports requestAnimationFrame. In the above code snippet on Chromium browsers, the first web worker is always able to run requestAnimationFrame successfully while its nested web worker fails, thus contradicting the spec.

### Do you have anything more you want to share?

_No response_

### MDN URL

_No response_

### MDN metadata

_No response_

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Summary

`DedicatedWorkerGlobalScope.requestAnimationFrame()` is not supported in nested Web Workers on chromium

see test & detail at https://github.com/mdn/content/pull/30880#discussion_r1422377479 & https://github.com/mdn/content/pull/30880#discussion_r1422009358, also see the browser issue https://issues.chromium.org/issues/41483010, and the issue later fired https://github.com/mdn/browser-compat-data/issues/21964

add the note as the issue seems not going to be fixed recently

Note: I'm not sure if partial_implementation or impl_url key should be added

#### Related issues

Fixes #21964 
/cc @ArhanChaudhary

### CODE REVIEW

File: api/DedicatedWorkerGlobalScope.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "69"
+              "version_added": "69",
+              "notes": "Not supported in nested Web Workers. See <a href='https://crbug.com/41483010'>bug 41483010</a>."
```
Comment (skyclouds2001) [inline api/DedicatedWorkerGlobalScope.json:58]:
```suggestion
              "version_added": "69"
```

File: api/DedicatedWorkerGlobalScope.json [hunk 2]
```diff
@@ -358,7 +358,8 @@
...
```
Comment (caugner) [inline api/DedicatedWorkerGlobalScope.json:362]: Let's move this into a [`worker_support`](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#web-workers-worker_support) sub-feature.

**Review by caugner — CHANGES_REQUESTED** [review]

File: api/DedicatedWorkerGlobalScope.json (hunk 2 above)
Comment (skyclouds2001) [reply api/DedicatedWorkerGlobalScope.json:362]: I' m not quite sure here, as the feature itself is worker support **_only_**.

It is quite unnecessary to add one extra [worker_support](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#web-workers-worker_support) sub-feature.

Comment (caugner) [reply api/DedicatedWorkerGlobalScope.json:362]: Good point, I take my suggestion back.

**Review by caugner — APPROVED** [review]

**Pull request merged**